packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -359,6 +359,11 @@
   background-color: var(--notifi-placeholder-color);
 }
 
+.NotifiIntercomSendMessageSection__button:disabled
+{
```

Comment:
Can you please put `NotifiIntercomSendMessageSection__button:disabled {` on one line?